UnityProject/Assets/Scripts/HealthV2/Living/LivingHealthMasterBase.cs

```diff
@@ -793,4 +793,28 @@ public virtual string GetExamineText()
 	{
 		return "Weeee";
 	}
+
+	public bool HasBodyPart(BodyPartType bodyPartType, bool surfaceOnly = true)
+	{
+		foreach (var container in RootBodyPartContainers)
+		{
+			if (container.bodyPartType == bodyPartType)
+			{
+				if (surfaceOnly)
+				{
+					return true;
+				}
+
+				foreach (var bodyPart in container.ContainsLimbs)
+				{
+					if (bodyPart.bodyPartType == bodyPartType)
+					{
+						return true;
+					}
+				}
+			}
+		}
+
+		return false;
+	}
```

Comment:
This can be removed bod added one in https://github.com/unitystation/unitystation/pull/6210